### ISSUE TITLE
Delegating modules - eclipse integration

### ISSUE THREAD

**Issue by nicowiggersiv** [beryx/badass-jlink-plugin#151]
I am using the eclipse plugin in gradle and use vscode with Redhats java extension, which is using the eclipse JDT. In this setup JDT complains about the plain JAR module names, which it cannot resolve as a module (of course it can not resolve these modules, as these are generated delegating modules from the jlink plugin). As a result, all packages from within the plain jars, are shown as problems ("not accessible") as JDT can not find these packages in the resolved java modules. I guess eclipse JDT users should have the same issue.

Is this issue addressed somehow yet?

**Comment by siordache** [comment]
The plugin has no support for Eclipse JDT. I tried to reproduce your issue using one of my projects and I ran into a different problem. In my case, VSCode showed "xxx cannot be resolved to a module" for all automatic modules required in my `module-info.java`. I was able to get rid of this error by using [this workaround](https://github.com/eclipse/eclipse.jdt.ls/issues/957#issuecomment-472671834). Maybe it also works in your case.
